notebooks/240-yolox-object-detection/240-yolox-object-detection.ipynb

```diff
@@ -0,0 +1,4585 @@
+{
```

Comment:
<p>in some cases it can be useful to  provide model performance on small subset without accuracy testing on whole dataset, could you please add step for benchmarking model performance as separated step from accuracy evaluation?</p><p>Users may want to see performance benefit from quantization without long lasting accuracy checking. you can use benchmark_app for these purpoces</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/845/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='60'/>

Reply:
`benchmark_App` testing added. Looking forward to any other suggestion you might have. @eaidova

Reply:
@eaidova There is some bug in `227-whisper-subtitles-generation` notebook that is failing ci tests, as you might know already. For example you can see my *nbval / build_nbval (ubuntu-20.04, 3.8) (pull_request)* test is stuck, that has cancelled other tests.

Without all CI tests running is there any other way to know if there was any CI test triggering error in my own notebook, as I am going through incremental changes in my notebook? 

And as @adrianboguszewski  has mentioned,  as a participants all my CI checks should be passed. 
- [ ] all CI checks passed

I hope a solution or clarification might be helpful for the GSOC'23 participants. Thank you.